addons/hr_recruitment/views/hr_job_views.xml

```diff
@@ -99,8 +99,20 @@
                                 <div t-if="record.application_count.raw_value > 0">
                                     <field name="application_count"/> Applications
                                 </div>
-                                <a t-if="record.activities_today.raw_value > 0" name="action_open_today_activities" type="object" class="text-warning"><field name="activities_today"/> Activities Today</a>
-                                <a t-if="record.activities_overdue.raw_value > 0" name="action_open_late_activities" type="object" class="text-danger"><field name="activities_overdue"/> Late Activities</a>
+                                <div
+                                    t-if="record.activities_today.raw_value > 0"
+                                    name="action_open_today_activities"
+                                    type="object"
+                                    class="text-warning">
+                                    <field name="activities_today"/> Activities Today
+                                </div>
+                                <div
+                                    t-if="record.activities_overdue.raw_value > 0"
+                                    name="action_open_late_activities"
+                                    type="object"
+                                    class="text-danger">
+                                    <field name="activities_overdue"/> Late Activities
+                                </div>
```

Comment:
not really a good change on an accessibility level to use `div` instead of `a` tags :thinking: Couldn't the issue be solved with just a `<br/>` tag between both tags?

Reply:
you're right :D